shell/src/main/java/alluxio/cli/fsadmin/report/SummaryCommand.java

```diff
@@ -88,6 +89,21 @@ private void printMetaMasterInfo() throws IOException {
     print("Uptime: " + CommonUtils.convertMsToClockTime(masterInfo.getUpTimeMs()));
     print("Version: " + masterInfo.getVersion());
     print("Safe Mode: " + masterInfo.isSafeMode());
+
+    boolean isZookeeperEnabled = masterInfo.isZookeeperEnabled();
+    print("Zookeeper Enabled: " + isZookeeperEnabled);
+    if (isZookeeperEnabled) {
+      String zookeeperAddress = masterInfo.getZookeeperAddress();
+      if (zookeeperAddress != null) {
+        String[] zkAddressesArray = zookeeperAddress.split(",");
```

Comment:
if the client side is going to use the zk addresses as a list like this, the server should expose them as a `List<String>` instead of a raw `String`. It enforces that the client and server agree on the basic structure of zookeeperAddress (maybe rename to zookeeperAddresses?). We can either do the parsing server-side, or have the client treat the addresses as a raw string and display it directly

Reply:
Good point! Even me, I think of the zk addresses format for some time and go to the alluxio website to figure out the possible formats. Export it as a list of string is more understandable. Changed, thanks!